aikau/src/main/resources/alfresco/lists/AlfSortablePaginatedList.js

```diff
@@ -240,7 +240,7 @@ define(["dojo/_base/declare",
                }
                else
                {
-                  this.loadData();
+                  this.onReloadData();
```

Comment:
Is this reacting to a reloadData event, or requesting a data-reload? Just querying the new prefix.